src/views/Logbook/LogbookList.tsx

```diff
@@ -20,13 +21,119 @@ import { analytics } from "~/utils"
 
 import * as styles from "./LogbookList.module.css"
 
-const LogbookList = () => {
-  const { locale } = useLocalization()
+type ItemProps = {
+  logbook: Logbook
+}
+
+const LogbookListContentItem: React.FC<ItemProps> = ({ logbook }) => {
   const isMediumUp = useResponsive("md-up")
   const iconSize = isMediumUp ? "xl" : "xlM"
+
+  return (
+    <div className={styles.listitem}>
+      <div
+        className={styles.avatar}
+        style={{
+          backgroundImage: `url("${logbook.tokenImageURL}")`,
+        }}
+      />
+      <div className={styles.avatarDescription}>
+        <h3>Traveloggers #{logbook.tokenId}</h3>
+        <p>Collectors 9 / Edited {logbook.logs.length}</p>
```

Comment:
Collector count and Etherscan link are not implemented yet. In previous discussion, we plan to use event (`LogbookNewLog`) filtering, but seems didn't work, will try again later.

cc @Carolusian